docs/api/en/helpers/CameraHelper.html

```diff
@@ -12,40 +12,35 @@
 		<h1>[name]</h1>
 
 		<p class="desc">
-		This helps with visualizing what a camera contains in its frustum.<br />
-		It visualizes the frustum of a camera using a [page:LineSegments].
+			This helps with visualizing what a camera contains in its frustum.<br />
+			It visualizes the frustum of a camera using a [page:LineSegments].
 		</p>
 
 		<h2>Code Example</h2>
 		<code>
-const camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 1000 );
-const helper = new THREE.CameraHelper( camera );
-scene.add( helper );
+			const camera = new THREE.PerspectiveCamera( 75, window.innerWidth /
+			window.innerHeight, 0.1, 1000 ); const helper = new THREE.CameraHelper(
+			camera ); scene.add( helper );
```

Comment:
Please undo these changes. The code in this block should be like that. Otherwise the formatting will be odd.

https://raw.githack.com/servinlp/three.js/feature/dispose-docs/docs/index.html#api/en/helpers/CameraHelper

Reply:
In general, please avoid any other formatting changes. It's best to focus on just the return types.

Reply:
Sorry about that, though I had it turned off. Will correct it.